test/api/profileTypes.test.js

```diff
@@ -1,33 +1,318 @@
 import 'test/enzyme-init';
-import { getProfileTypes } from 'api/profileTypes';
 import {
+  postProfileType,
+  putProfileType,
+  deleteProfileType,
+  getProfileType,
+  getProfileTypes,
+  getAttributeFromProfileType,
+  deleteAttributeFromProfileType,
+  postAttributeFromProfileType,
+  putAttributeFromProfileType,
+  getProfileTypeAttributes,
+  getProfileTypeAttribute,
+} from 'api/profileTypes';
+
+import { makeRequest, METHODS } from '@entando/apimanager';
+
+import {
+  PROFILE_TYPES,
+  PROFILE_TYPES_DELETE_OK,
+  ATTRIBUTE_PROFILE_TYPES_DELETE_OK,
   PROFILE_TYPES_OK_PAGE_1,
-  PROFILE_TYPES_OK_PAGE_2,
+  PROFILE_TYPES_ATTRIBUTES,
+  PROFILE_TYPE_ATTRIBUTE,
 } from 'test/mocks/profileTypes';
 
+const correctRequest = {
+  uri: '/api/profileTypes',
+  method: METHODS.GET,
+  mockResponse: PROFILE_TYPES_OK_PAGE_1,
+  useAuthentication: true,
+};
+
+
 jest.unmock('api/profileTypes');
-jest.useFakeTimers();
+jest.mock('@entando/apimanager', () => ({
```

Comment:
This is mocked globally already, please remove and adjust the global mock if necessary